database/seeders/DatabaseSeeder.php

```diff
@@ -2,22 +2,51 @@
 
 namespace Database\Seeders;
 
+use App\enums\Role;
+use App\Models\Channel;
+use App\Models\Guild;
+use App\Models\Message;
 use App\Models\User;
-// use Illuminate\Database\Console\Seeds\WithoutModelEvents;
 use Illuminate\Database\Seeder;
 
 class DatabaseSeeder extends Seeder
 {
-    /**
-     * Seed the application's database.
-     */
     public function run(): void
     {
-        // User::factory(10)->create();
+        $admin = User::factory()->create([
```

Comment:
Maybe a config('app.env') == 'local' would fit better here, just to avoid confusion when running in production.